### PR TITLE
restats/balances/names/sprites the T-45/M4RA

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Renames it to scout rifle rather than battle rifle, less confusion
Renames it to TX-8
gives it a nifty sprite by kuro
image

Changes
+Firerate increased to 0.4 from .45 that rounds to 0.5ish
+5 more in a magazine for 25 total
+More attachment options
+Doesn't look like a bad block
-Higher wield slowdown
-Less out of the box accuracy

## Why It's Good For The Game
The M4RA is hot trash and hopefully this makes it less hot trash, atleast it'll have a gimmick and is worth the 50 points

## Changelog
:cl:
add:new tx-8, aka the t-45 but resprited/named
del:t-45
balance: compared to t-45 the tx-8 fires slightly faster and holds more in a magazine, has more attachment options
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/marine_boxes.dm [hunk 1]
```diff
@@ -385,7 +385,7 @@
 
 /obj/item/storage/box/spec/scout
 	name = "\improper Scout equipment"
-	desc = "A large case containing Scout equipment; this one features the M4RA battle rifle. Drag this sprite into you to open it up!\nNOTE: You cannot put items back inside this case."
+	desc = "A large case containing Scout equipment; this one features the tx8 battle rifle. Drag this sprite into you to open it up!\nNOTE: You cannot put items back inside this case."
```
Comment (Pariah919) [inline code/game/objects/items/storage/marine_boxes.dm:388]:
```suggestion
	desc = "A large case containing Scout equipment; this one features the TX-8 battle rifle. Drag this sprite into you to open it up!\nNOTE: You cannot put items back inside this case."
```

**Review by Pariah919 — COMMENTED** [review]
mmissed one

**Review by TiviPlus — APPROVED** [review]
Imo looks like bad but thats for bimmer to decide

**Review by TheBimmer231 — APPROVED** [review]

File: code/modules/clothing/glasses/night.dm [hunk 2]
```diff
@@ -10,9 +10,9 @@
 	lighting_alpha = LIGHTING_PLANE_ALPHA_INVISIBLE
 
 
-/obj/item/clothing/glasses/night/M4RA
-	name = "\improper M4RA Battle sight"
-	desc = "A headset and night vision goggles system for the M4RA Battle Rifle. Allows highlighted imaging of surroundings. Click it to toggle."
+/obj/item/clothing/glasses/night/tx8
+	name = "\improper TX-8 Battle sight"
```
Comment (XSlayer300) [inline code/modules/clothing/glasses/night.dm:14]:
```suggestion
	name = "\improper TX-8 battle sight"
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**